sos/report/plugins/hbn.py

```diff
@@ -12,13 +12,28 @@
 
 class HBN(Plugin, IndependentPlugin):
 
-    short_desc = 'HBN (Host Based Network)'
+    short_desc = 'HBN (Host Based Networking)'
     plugin_name = "hbn"
+    vsctl = "ovs-vsctl"
     packages = ('hbn-repo',)
 
     def setup(self):
         self.add_copy_spec([
             '/etc/mellanox',
             '/var/log/doca/hbn',
             '/var/lib/hbn/etc/cumulus',
+            '/var/log/containers',
+            '/var/log/sfc-install.log',
+            '/tmp/sf_devices',
+            '/tmp/sfr_devices',
+            '/tmp/sfc-activated',
+        ])
+
+        self.add_journal(units="sfc")
+        self.add_journal(units="sfc-state-propagation")
+
+        self.add_cmd_output([
+            "crictl ps -a",
```

Comment:
same as above this is captured by the container plugin + other things

Reply:
ok, I will drop it